clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -811,10 +816,11 @@ void ContinuationIndenter::addTokenOnCurrentLine(LineState &State, bool DryRun,
     if (!Tok.Previous)
       return true;
     if (Tok.Previous->isIf())
-      return Style.AlignAfterOpenBracket == FormatStyle::BAS_AlwaysBreak;
-    return !Tok.Previous->isOneOf(TT_CastRParen, tok::kw_for, tok::kw_while,
-                                  tok::kw_switch) &&
-           !(Style.isJavaScript() && Tok.Previous->is(Keywords.kw_await));
+      return Style.AlignAfterOpenBracketBreak.InIfConditionalStatements;
```

Comment:
Don't you change the behavior here?

Reply:
It's been refactored into the `IsOtherConditional()`. I may need to look a bit closer at the JavaScript (TypeScript) cases.